test/test-cases/typescript-css-modules/typescript-css-modules.test.js

```diff
@@ -65,7 +65,7 @@ describe('typescript-css-modules', () => {
     });
 
     it('should generate the expected files', async () => {
-      const files = await dirContentsToObject(distDir);
+      const files = await dirContentsToObject(distDir, ['.js', '.css']);
```

Comment:
Why is this being limited to JS and CSS now?

Reply:
We now spit out a `webpackStats.json` file which contains info that will be specific to the machine that created it so it's not a good candidate for snapshots. The function currently only has a whitelist so I just whitelisted everything else.

Reply:
Do we only output this file for server rendering?

Reply:
Yes

Reply:
😎